### PR TITLE
mgmt upgrade network and compute

### PR DESCRIPTION
**Pull request by xseeseesee**
1. update network to `2020-07-01`
2. update compute to `2020-06-01`
3. update changelogs
4. run live tests and update session records

### CODE REVIEW

File: sdk/resourcemanager/azure-resourcemanager-compute/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 ## 2.2.0-beta.1 (Unreleased)
 
+- Updated `api-version` to `2020-06-01`
```
Comment (ChenTanyi) [inline sdk/resourcemanager/azure-resourcemanager-compute/CHANGELOG.md:5]: Is it `2020-06-30`?

**Review by ChenTanyi — COMMENTED** [review]
Please also add important breaking change in the change log, like `java.field.removed`, `java.method.returnTypeChanged`, `java.method.parameterTypeChanged`

File: sdk/resourcemanager/azure-resourcemanager-compute/CHANGELOG.md (hunk 1 above)
Comment (xccc-msft) [reply sdk/resourcemanager/azure-resourcemanager-compute/CHANGELOG.md:5]: The api version of compute is `2020-06-01`. The package is `2020-06-01`.

**Comment by weidongxu-microsoft** [comment]
Which commits contains major changes?

**Comment by xccc-msft** [comment]
> Which commits contains major changes?

[fix network compile errors](https://github.com/Azure/azure-sdk-for-java/pull/18386/commits/03e9b85ad39a389b3b9cef5f7c4cd83d4fd55082)
[fix compute compile errors](https://github.com/Azure/azure-sdk-for-java/pull/18386/commits/9a661d174ed1764a96796804e06c11496a098bbf)
[update revapi for network](https://github.com/Azure/azure-sdk-for-java/pull/18386/commits/469a9e50e87cf1936a4d62793aee869af55761f7)

I might revert revapi after code merge.

**Review by weidongxu-microsoft — APPROVED** [review]
Please fill the changelog.

**Pull request merged**